src/extensions/default/brackets-show-whitespace/styles/main.less

```diff
@@ -0,0 +1,76 @@
+/*
```

Comment:
Let's convert this to a `.css` file and modify the code above that loads it.  We dont' want to render LESS files in the browser.